lib/templates/updateDeviceLax.json

```diff
@@ -154,6 +154,21 @@
     "static_attributes": {
       "description": "Free form array of data to be appended to the target entity",
       "type": "array"
+    },
+    "timestamp": {
+      "description": "Timestamp",
+      "type": "boolean"
+    },
+    "expressionLanguage": {
+      "description": "Expression language used to apply expressions for this device",
+      "type": "string"
+    },
```

Comment:
```suggestion
```

(Maybe appeared here because we forget remove it in the config grup equivalente template?)